config/openapitools.json

```
@@ -65,6 +65,12 @@
           "packageVersion": "1.0.0-alpha.27"
         }
       },
+      "javascript-ingestion": {
+        "output": "#{cwd}/clients/algoliasearch-client-javascript/packages/ingestion",
+        "additionalProperties": {
+          "packageVersion": "1.0.0-alpha"
```

```suggestion
          "packageVersion": "1.0.0-alpha.1"
```

we've started at .1 for all the other packages